app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,59 @@
+class Admin::UsersController < ApplicationController
+  
+  before_action :require_admin
+	before_action :set_user, only: [:show, :edit, :update, :destroy]
```

Comment:
`%i`記法を使うとスッキリ記載できます。
```suggestion
  before_action :set_user, only: %i[show edit update destroy]
```